src/UI/class-recommended-widget.php

```diff
@@ -109,27 +139,26 @@ public function widget( $args, $instance ): void {
 		echo wp_kses_post( $title_html );
 
 		// Set up the variables.
-		$options = $this->parsely->get_options();
 		$api_url = $this->get_api_url(
-			$options['apikey'],
+			$this->parsely->get_api_key(),
 			$instance['published_within'],
 			$instance['sort'],
 			$instance['boost'],
-			(int) $instance['return_limit']
```

Comment:
No need as our defined array strucutre already knows that `return_limit` is of type `int`